loglist/loglist.go

```diff
@@ -142,10 +142,26 @@ func (ll *LogList) FindLogByKeyHash(keyhash [sha256.Size]byte) *Log {
 	return nil
 }
 
+// FindLogByKeyHashPrefix finds all logs whose key hash starts with the prefix.
+func (ll *LogList) FindLogByKeyHashPrefix(prefix []byte) []*Log {
+	var results []*Log
+	if len(prefix) > sha256.Size {
+		return results
```

Comment:
nit: it might be slightly clearer what to expect from this func in this situation if you explicitly return 'nil' here and move the var down to just before the loop where it's needed?

Reply:
Done.